src/hooks/screen/useChatScreenHook.ts

```diff
@@ -174,8 +174,8 @@ function useChatScreenHook(type: 'SIGNED' | 'ANONYMOUS'): UseChatScreenHook {
         response = await AnonymousMessageRepo.sendAnonymousMessage(
           selectedChannel?.id,
           message,
-          replyData?.id,
-          newAttachments
+          newAttachments,
+          replyData?.id
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change in the order of arguments passed to the `sendAnonymousMessage` function seems fine as long as the function definition has been updated accordingly. If not, this could lead to incorrect behavior or errors. Please ensure that the function definition for `sendAnonymousMessage` is updated to match this new order of arguments.
```diff
-        response = await AnonymousMessageRepo.sendAnonymousMessage(
-          selectedChannel?.id,
-          message,
-          replyData?.id,
-          newAttachments
-        );
+        response = await AnonymousMessageRepo.sendAnonymousMessage(
+          selectedChannel?.id,
+          message,
+          newAttachments,
+          replyData?.id
+        );
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->